inbc-program/inbc/command/config_command.py

```diff
@@ -46,7 +46,10 @@ def search_response(self, payload: str) -> None:
         if search_keyword(payload, CONFIGURATION_SUCCESSFUL_MESSAGE_LIST):
             self.terminate_operation(COMMAND_SUCCESS, InbcCode.SUCCESS.value)
         elif search_keyword(payload, CONFIGURATION_FAILURE_MESSAGE_LIST):
-            self.terminate_operation(COMMAND_FAIL, InbcCode.FAIL.value)
+            if search_keyword(payload, ["ERROR! No eligible nodes found"]):
```

Comment:
Place this string as a constant in the inbm_common_lib/request_message_constants.py file.  Then change the 2 places in INBC where that string is used with the constant.  Also change the place where that string is sent from in the Vision-agent.